entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+source /opt/ros/galactic/setup.bash
+
```

Comment:
This doesn't do anything after sourcing, and it's not used by the built container image, although it's referenced from Dockerfile.build_env

Reply:
Removed not used entrypoint script.